frontend/src/state/restInterfaces.ts

```diff
@@ -151,6 +151,7 @@ export enum CompressionType {
 
 export interface Payload {
     payload: any, // json obj
+    isPayloadNull: boolean,
```

Comment:
is this actually getting populated by the backend?

Reply:
Yes, backend code change is also part of this PR